dds/DCPS/XTypes/TypeAssignability.h

```diff
@@ -119,7 +85,20 @@ class OpenDDS_Dcps_Export TypeAssignability {
   bool get_union_member(const MinimalTypeObject*& ret, const CommonStructMember& m) const;
 
 private:
-  TypeLookup typelookup_;
+  TypeLookupService* tl_service;
+
+  const MinimalTypeObject& lookup_minimal(const TypeIdentifier& ti) const
+  {
+    return tl_service->GetTypeObject(ti).minimal;
+  }
+
+public:
+  void insert_entry(const TypeIdentifier& ti, const TypeObject& tobj)
+  {
+    tl_service->AddTypeObjectsToCache(ti, tobj);
+  }
+
+  static std::map<unsigned int, MinimalTypeObject> table_;
```

Comment:
`table_` can be removed. It served to emulate the cache for `TypeIdentifier` to `TypeObject` mapping, but now we have a real one.